### PR TITLE
Video Intelligence: Add new autogen layer.

### PR DESCRIPTION
**Pull request by lukesneeringer**
Uses #3975 as a base.

This adds an updated auto-gen layer.
*Note:* The unit tests seem to fail. Requires investigation.

### CODE REVIEW

File: videointelligence/google/cloud/videointelligence_v1beta1/gapic/video_intelligence_service_client.py [hunk 1]
```diff
@@ -88,6 +88,8 @@ def __init__(self,
                 client library metrics. Ultimately serializes to a string
                 (e.g. 'foo/1.2.3 bar/3.14.1'). This argument should be
                 considered private.
+
+        Returns: VideoIntelligenceServiceClient
```
Comment (dhermes) [inline videointelligence/google/cloud/videointelligence_v1beta1/gapic/video_intelligence_service_client.py]: @lukesneeringer I thought you were anti-"returns in `__init__`"?

Comment (lukesneeringer) [reply videointelligence/google/cloud/videointelligence_v1beta1/gapic/video_intelligence_service_client.py]: I am. :-) I could have sworn I removed that.

Comment (lukesneeringer) [reply videointelligence/google/cloud/videointelligence_v1beta1/gapic/video_intelligence_service_client.py]: Yeah, I definitely got a old version or something somewhere. Fixing.

Comment (dhermes) [reply videointelligence/google/cloud/videointelligence_v1beta1/gapic/video_intelligence_service_client.py]: Canary in the GitHub mine

Comment (lukesneeringer) [reply videointelligence/google/cloud/videointelligence_v1beta1/gapic/video_intelligence_service_client.py]: Finally got new stuff built and fix pushed.

**Comment by dhermes** [comment]
Still LGTM

**Comment by lukesneeringer** [comment]
Waiting on Circle, then merging this.

**Pull request merged**